src/www/index.html

```diff
@@ -280,7 +301,7 @@
       <div class="col-md-4 col-xs-2"></div>
       <div class="col-md-offset-3 col-xs-offset-1 col-xs-10 col-md-6" style="margin-top: -20px;">
         <center>
-          <div id="status" style="margin-bottom:10px;"></div><a id="aLog" style="cursor:pointer;">-Logs-</a>
+          <div id="status" style="margin-bottom:10px;"></div><a id="aLog" style="cursor:pointer;"><font color="#000000">-Logs-</font>/a>
```

Comment:
@samyakbvs Can you please remove the inline css from here? 

Also for colours you can define a class and add it to the anchor tag instead of using font tag.